src/main/java/com/scalar/admin/AdminCommand.java

```diff
@@ -35,9 +35,7 @@
 
   @CommandLine.Option(
       names = {"--wait-outstanding", "-w"},
-      required = false,
-      defaultValue = "true",
-      paramLabel = "WAIT_OUTSTANDING",
+      defaultValue = "false",
```

Comment:
I'm a bit confused.
Changing the default value would fix the behavior?
Isn't it a server-side problem?

Reply:
Thank you for taking a look at this!
As discussed, I renamed the option and did some fixes related to it.